prow/istio-pilot-multicluster-e2e.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+# Copyright 2017 Istio Authors
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+# set this to the new multicluster-e2e type
+RESOURCE_TYPE="${RESOURCE_TYPE:-gke-e2e-test}"
+OWNER=istio-pilot-multicluster-e2e
+
+source "prow/istio-pilot-e2e-common.sh"
+
+# setup cluster-registries dir setup by mason
+CLUSTERREG_DIR=${CLUSTERREG_DIR:-$HOME/.kube}
+
+
+# These IP params don't really matter
+PilotIP="1.1.1.1"
+ClientCidr=1.1.1.0/24
+ServerEndpointIP=1.1.1.1
+
+PilotCfgStore=True
+
+# mason dumps all the kubeconfigs into the same file but we need to use per cluster
+# files for the clusterregsitry config.  Create the separate files.
+#  -- if PILOT_CLUSTER not set, assume pilot install to be in the first cluster
+unset IFS
```

Comment:
Can we extract this into a standalone file ? "bin/splitKubeConfig.sh" or something. 
It can be used to convert a kubeconfig with multiple clusters (quite common) to the configs needed to setup mc.

Reply:
I assume the script can later be modified to setup the secrets (when the second PR gets in)